### DISCUSSION TITLE
Do I need relay server?

### DISCUSSION THREAD

**Discussion by truthsword** [rustdesk/rustdesk#4447] [General]
New user here... self-hosted on Docker. Use is limited within LAN.

What is the functional difference between the 2 RustDesk containers? Since I don't use this externally, do I need the relay server? Would using direct IP make a difference?

Just curious!

**Comment by SergeyMy** [comment]
https://github.com/rustdesk/rustdesk/discussions/4380
2. How does RustDesk work
https://github.com/rustdesk/rustdesk/wiki/How-does-RustDesk-work%3F

_Do you need a relay server?_
will never be redundant :)
